studio-ui/static-assets/themes/cstudioTheme/css/contextNav.css

```
@@ -0,0 +1,1165 @@
+	letter-spacing: normal;
+	line-height: normal;
+	text-align: left;
+	text-indent: 0;
+	text-justify: auto;
+	text-outline: none;
+	text-shadow: none;
+	text-transform: none;
+	white-space: normal;
+	word-spacing: normal;
+	word-break: normal;
+	word-wrap: normal;
```

Stylelint (17.14.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)